src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/OsxIPv4GlobalStatistics.cs

```diff
@@ -51,12 +50,12 @@ public unsafe OsxIPv4GlobalStatistics()
             _defaultTtl = statistics.DefaultTtl;
             _forwarding = statistics.Forwarding == 1;
 
-            HashSet<string> interfaceSet = new HashSet<string>();
+            Dictionary<string, object> interfaceSet = new Dictionary<string, object>();
```

Comment:
Don't do this (and similar) we can fix it with reference assemblies in the build

Reply:
So we can reference `System.Core` from `System`?